bamboo/set-bamboo-env-variables.sh

```diff
@@ -35,15 +35,41 @@ done
 
 export COMMIT_MSG=$(git log --pretty='format:%Creset%s' -1)
 export GIT_SHA=$(git rev-parse HEAD)
+
+## This should take a blank value from the global options, and
+## is intended to allow an override for a custom branch build.
+export GIT_PR=$bamboo_GIT_PR
```

Comment:
GIT_PR can be overridden at the branch trigger level.  This allows for manual builds of branch build plans that don't have a PR.